packages/commonwealth/client/scripts/views/pages/CommunityManagement/Integrations/Stake/validation.ts

```diff
@@ -0,0 +1,19 @@
+import z from 'zod';
+
+const snapshotNameSchema = z.string().regex(/^[a-zA-Z0-9-.]+\.((xyz)|(eth))$/, {
+  message: 'Snapshot must be valid, and end in *.eth or *.xyz',
+});
+const snapshotLinkSchema = z
+  .string()
+  .regex(
+    /^https:\/\/(\w+\.)?snapshot\.org\/#\/[a-zA-Z0-9-.]+\.((xyz)|(eth))$/,
+    {
+      message: 'Snapshot link be valid, and end in *.eth or *.xyz',
```

Comment:
`must` missing? Is this copy provided by the design/product team?

Reply:
this file shouldn't be there, deleted it.